andela/activo-web
# #158564874 Set fields to required or unique.

**What does this PR do?**
Create UI functionality for making a field required or unique.

**Description of Task to be completed?**
- create checkbox component
- write styles for checkbox component
- write method handlers to listen for click events on the checkbox and update state
- write tests for checkbox component

**How should this be manually tested?**
After cloning and running app, go to the category page and click on one of the additional attributes select box. 
- The background should now be greyd out and buttons appear on the top right.
- Click on the little green button.
- Click on the check boxes to toggle it between being checked and not checked.

**Any background context you want to provide?**
N/A

**What are the relevant pivotal tracker stories?**
[#158564874](https://www.pivotaltracker.com/story/show/158564874)

**Screenshots (if appropriate)**
<img width="1280" alt="screen shot 2018-07-13 at 2 22 59 pm" src="https://user-images.githubusercontent.com/24798364/42694034-e43320a4-86a8-11e8-8b4b-2d1c2f4f3d51.png">

<img width="1280" alt="screen shot 2018-07-13 at 2 23 02 pm" src="https://user-images.githubusercontent.com/24798364/42694048-f20485a6-86a8-11e8-89bb-2a736fb03a25.png">


**Questions:**

N/A
